.github/CONTRIBUTING.md

```diff
@@ -171,7 +171,7 @@ Run these commands
 git submodule update --init --recursive
 pip install -r requirements/docs.txt
 cd docs
-make html
+make html  # You could disable copying notebooks by running `SHOULD_COPY_NOTEBOOKS=0 make html`
```

Comment:
is this working for you? for me it isn't. can you check?

Reply:
Should work now!